app/styles/style.css

```diff
@@ -131,4 +137,289 @@ body {
     text-decoration: none;
     display: flex;
   }
+
+  .jobs {
+    padding: 24px;
+  }
+
+  .card {
+    display: flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    flex-direction: column;
+    height: auto;
+    width: 100%;
+    background-color: #fff;
+    border-radius: 16px;
+    border: 1px #dfe1e6 solid;
+    margin-top: 114px;
+  }
```

Comment:
- `[OPTIONAL]` Kindly try to make your design as much as closer to the `Figma` design because it is important to carefully look at the small details which indeed make you not only a better developer but also make you a `detail-oriented` developer. I noticed that you are using `margin-top: 114px` but I please use `88px`, because your design have gaps more than required which makes your design looks bad.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-04-29 13-12-38](https://user-images.githubusercontent.com/66476329/235292564-6441af33-f5d7-40b4-b843-ee02054a1aae.png)

</td>
<td>

![Screenshot from 2023-04-29 13-21-53](https://user-images.githubusercontent.com/66476329/235292553-13cc15fd-52d9-4001-9eb4-f11add307607.png)

</td>
</tr>
</table>